### ISSUE TITLE
Search results missing

### ISSUE THREAD

**Issue by DawidCh** [Requarks/wiki#50]
Hello.
I have only one user, which is not Administrator, who can use serach. For other, as well as, Administrator, roles (Global Administrator, User) search returns no results. Logs are clean. What may be the problem?
Regards, Dawid.
PS. Beta9 is current master, am I right?

**Comment by NGPixel** [comment]
Do you have any errors in the browser dev console (F12) when performing a search?

**Comment by DawidCh** [comment]
JS console in browser is clean as well as logs at server side.

**Comment by DawidCh** [comment]
I have installed new release. Cool wizard;)
I have tested the search a bit and I'm able to get results when I'm looking for a word which is contained in:
```
<!-- TITLE: Galeria -->
<!-- SUBTITLE: A quick summary of Galeria which helps you display images -->
```
If I'm looking for a word from content of the page I get 0 results.
Console server logs are clean. In documentation I have found that it also should search in the content of the page, right?

**Comment by NGPixel** [comment]
There's indeed a bug with some pages content not being indexed properly.

**Comment by NGPixel** [comment]
Fixed for Beta 12 (f5aa726cd54fd96b003d79441f05cd0099728c2c)
